src/main/java/org/assertj/core/util/Throwables.java

```diff
@@ -148,6 +150,21 @@ public static Throwable getRootCause(Throwable throwable) {
     return throwable;
   }
 
+  /**
+   * Get the causes chain of a {@link Throwable}.
+   *
+   * @param throwable the {@code Throwable} to get causes chain from.
+   * @return the causes chain, including the {@link Throwable} provided as parameter.
+   */
+  public static List<Throwable> getCausesChain(Throwable throwable) {
```

Comment:
I don't see the need of exposing this as a public API, can be a private method of `AbstractThrowableAssert`.

Reply:
Agreed, we'll move it here if we reuse that piece of code elsewhere.